### ISSUE TITLE
Add option to ignore DNS for Let's Encrypt per domain

### ISSUE THREAD

**Issue by lukasbableck** [Froxlor/Froxlor#1170]
**Is your feature request related to a problem? Please describe.**
For some domains the IP address in Froxlor does not match the IP address in DNS (for example when you're using Cloudflare).
While the feature to check DNS before trying to create a certificate via Let's Encrypt is useful and definitely makes sense to have enabled, it would be great if there would be an option to ignore DNS for a specific domain.

**Describe the solution you'd like**
Additional option in the domain settings to disable the DNS check.

**Describe alternatives you've considered**
Disabling the DNS check completely, but that's not really a solution as I would like to leave that enabled

**Comment by envoyr** [comment]
Yes, we could add an option to the domain that skips the DNS check. 🤔

Alternatively, I can think of adding an HTTP challenge (/.well-known/acme-challenge/) verification in addition to the DNS validation.

So the domain could still be checked if it is configured correctly before it runs into a rate limit.
